frontend/packages/console-dynamic-plugin-sdk/docs/console-extensions.md

```diff
@@ -550,7 +550,7 @@ Gives full control over Console feature flags.
 
 | Name | Value Type | Optional | Description |
 | ---- | ---------- | -------- | ----------- |
-| `handler` | `CodeRef<FeatureFlagHandler>` | no | Used to set/unset arbitrary feature flags. |
+| `handler` | `CodeRef<(callback: SetFeatureFlag) => void>` | no | Used to set/unset arbitrary feature flags. |
```

Comment:
Note that the `(callback: SetFeatureFlag) => void` type comes from core extensions, i.e. the `SetFeatureFlag` symbol is not the one declared by Console, but the one declared by `@openshift/dynamic-plugin-sdk` package.